docs/installation.rst

```diff
@@ -93,6 +115,23 @@ For building your Client app in your host machine, you need to build against the
 
     gcc <your_main.c> -lmicrocdr -lmicroxrcedds_client
 
+.. _install_client_docker:
+
+Using the provided Docker image
+*******************************
+
+    The *eProsima Micro XRCE-DDS Client* comes with a `Docker image <https://hub.docker.com/r/eprosima/micro-xrce-dds-agent>`_ where the library is installed, as well as the provided examples, so users can easily execute them.
```

Comment:
where the library is installed together with the examples, so they can easily be executed by the users.